testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -932,6 +932,14 @@
   row.set(true)
 end
 
+When(/^I uncheck "([^"]*)" in the list$/) do |text|
+  top_level_xpath_query = "//div[@class=\"table-responsive\"]/table/tbody/tr[.//td[contains(.,'#{text}')]]//input[@type='checkbox']"
+  row = find(:xpath, top_level_xpath_query, match: :first)
+  raise "xpath: #{top_level_xpath_query} not found" if row.nil?
+
+  row.set(false)
+end
+
```

Comment:
Ah I see, we have one for check, but not for uncheck.
Ok, what about something like...

```suggestion
When(/^I (check|uncheck) "([^"]*)" in the list$/) do |check_option, text|
  top_level_xpath_query = "//div[@class=\"table-responsive\"]/table/tbody/tr[.//td[contains(.,'#{text}')]]//input[@type='checkbox']"
  row = find(:xpath, top_level_xpath_query, match: :first)
  raise "xpath: #{top_level_xpath_query} not found" if row.nil?

  row.set(check_option == 'check')
end

```